_config/site.yml

```diff
@@ -391,16 +397,16 @@ projects:
       java:
         name: Java
         url: https://www.oracle.com/technetwork/java/javase/downloads/index.html
-      bv:
+      java_ee_bv:
         name: Bean Validation
         url: https://beanvalidation.org/
-      ee:
+      java_ee:
         name: Java EE
         url: https://www.oracle.com/java/technologies/java-ee-glance.html
-      jbv:
+      jakarta_ee_bv:
         name: Jakarta Bean Validation
```

Comment:
What name should we use for this one ... since it got shorter with 3.1 (Jakarta Validation) ?

Reply:
I'll let you do that update, as I suspect it's not _just_ about this entry -- there are likely other places in the website where we use "Jakarta Bean Validation".

+1 for "Jakarta Validation", even for early "Jakarta Bean Validation" releases, even if that's not 100% correct.